staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -94,9 +93,10 @@ func TestRequestSetsHeaders(t *testing.T) {
 }
 
 func TestRequestWithErrorWontChange(t *testing.T) {
+	gvCopy := v1.SchemeGroupVersion
 	original := Request{
 		err:     errors.New("test"),
-		content: ContentConfig{GroupVersion: &api.Registry.GroupOrDie(api.GroupName).GroupVersion},
+		content: ContentConfig{GroupVersion: &gvCopy},
```

Comment:
There is no existing Registry in apimachinery.

IMO client should have a global test registry shared by all client tests. And if a client test needs aspecial registry, it creates its own.